content/blog/2021-01-26-january-21-community-gems.md

```diff
@@ -0,0 +1,184 @@
+---
+title: January '21 Community Gems
+date: 2021-01-26
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: parallelize your data transfer, 
+  compressed datasets, and DVC pipelines in CI/CD.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: parallelize your data transfer, 
+  compressed datasets, and DVC pipelines in CI/CD.
+picture: 2020-12-30/cover.png
```

Comment:
probably we need a new cover here

Reply:
btw, name sounds a bit too generic - probably should be something like `gems-cover`